### PR TITLE
Continuous Integration

### PR DESCRIPTION
**Pull request by dcuestam**
Make sure you have checked _all_ steps below.

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Benchmarks
- [x] My PR adds the following micro benchmarks __OR__ does not need benchmarks for this extremely good reason:

- [x] My commits have been squashed if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

## License
- [x] Add a [Mozilla Public License 2.0 license header](http://mozilla.org/MPL/2.0/) to all the new files.

### Documentation
- [x] In case of new functionality, my PR adds documentation that describes how to use it.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2018 Shapelets.io
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+language: python
```
Comment (avilchess) [inline .travis.yml]: This may not be need as you are running in docker

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+sudo: required
+python:
```
Comment (avilchess) [inline .travis.yml]: Idem here

File: README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+[![Documentation Status](https://readthedocs.org/projects/khiva-python/badge/?version=latest)](https://khiva-python.readthedocs.io/en/latest/?badge=latest)
+[![Build Status](https://travis-ci.com/shapelets/khiva-python.svg?branch=develop)](https://travis-ci.com/shapelets/khiva-python)
+[![codecov.io](https://codecov.io/github/shapelets/khiva-python/coverage.svg?branch=develop)](https://codecov.io/github/shapelets/khiva-python)
```
Comment (avilchess) [inline README.md]: If you want, you can add another for master, look at the following example:
https://github.com/conan-io/conan

File: .travis.yml (hunk 1 above)
Comment (dcuestam) [reply .travis.yml]: At the end, I am using python, I have changes the way I upload the coverage in order to not to publish the token.

File: .travis.yml (hunk 2 above)
Comment (dcuestam) [reply .travis.yml]: Idem answer

File: README.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
+
+[![Documentation Status](https://readthedocs.org/projects/khiva-python/badge/?version=latest)](https://khiva-python.readthedocs.io/en/latest/?badge=latest)
+[![License: MPL 2.0](https://img.shields.io/badge/License-MPL%202.0-brightgreen.svg)](https://github.com/shapelets/khiva-python/blob/master/LICENSE.txt)  
+[![py364 status](https://img.shields.io/badge/python3.6.4-supported-green.svg)](https://github.com/shapelets/khiva-python/issues)
+Build Master Branch [![Build Status](https://travis-ci.com/shapelets/khiva-python.svg?branch=master)](https://travis-ci.com/shapelets/khiva-python)
+Code Coverance Master Branch: [![codecov.io](https://codecov.io/github/shapelets/khiva-python/coverage.svg?branch=master)](https://codecov.io/github/shapelets/khiva-python)
+Build Develop Branch [![Build Status](https://travis-ci.com/shapelets/khiva-python.svg?branch=develop)](https://travis-ci.com/shapelets/khiva-python)
+Code Coverance Develop Branch [![codecov.io](https://codecov.io/github/shapelets/khiva-python/coverage.svg?branch=develop)](https://codecov.io/github/shapelets/khiva-python)
```
Comment (avilchess) [inline README.md]: Coverance or Coverage?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,13 @@
...
+
+
 # README #
 This is the KHIVA binding for connecting the Python programming language and the KHIVA library.
```
Comment (avilchess) [inline README.md]: This is the Khiva Python binding, it allows the usage of Khiva library from Python.

**Pull request merged**